packages/merkle_tree/src/merkle_tree.cairo

```diff
@@ -104,18 +106,18 @@ pub impl MerkleTreeImpl<T, +HasherTrait<T>, +Copy<T>, +Drop<T>> of MerkleTreeTra
     fn compute_root(
         ref self: MerkleTree<T>, mut current_node: felt252, mut proof: Span<felt252>
     ) -> felt252 {
-        while let Option::Some(proof_element) = proof
-            .pop_front() {
-                // Compute the hash of the current node and the current element of the proof.
-                // We need to check if the current node is smaller than the current element of the proof.
-                // If it is, we need to swap the order of the hash.
-                current_node =
-                    if Into::<felt252, u256>::into(current_node) < (*proof_element).into() {
-                        self.hasher.hash(current_node, *proof_element)
-                    } else {
-                        self.hasher.hash(*proof_element, current_node)
-                    }
-            };
+        while let Option::Some(proof_element) = proof.pop_front() {
```

Comment:
Can be modernized as `for proof_element in proof`

Reply:
Done